guardrails/llm_providers.py

```diff
@@ -53,14 +62,14 @@ def nonchat_prompt(prompt: str, instructions: Optional[str] = None) -> str:
 def chat_prompt(
     prompt: Optional[str],
     instructions: Optional[str] = None,
-    msg_history: Optional[List[Dict]] = None,
+    messages: Optional[List[Dict]] = None,
 ) -> List[Dict[str, str]]:
     """Prepare final prompt for chat engine."""
-    if msg_history:
-        return msg_history
+    if messages:
+        return messages
     if prompt is None:
         raise PromptCallableException(
-            "You must pass in either `text` or `msg_history` to `guard.__call__`."
+            "You must pass in either `text` or `messages` to `guard.__call__`."
         )
```

Comment:
Thinking a bit about the ergonomics of this method, will it complain if one calls `chat_prompt(messages=["...",])` and omits prompt?  It looks like there's no default value for prompt, so if it's unset we'll get weirdness (i.e., need to call `chat_prompt(prompt=None, messages=[...])`?  Should we initialize prompt to None?

Reply:
its a good question i wonder if i can kill this